lib/std/monotimes.nim

```diff
@@ -53,18 +57,14 @@ when defined(macosx):
 
 when defined(js):
   proc getJsTicks: float =
-    ## Returns ticks in the unit seconds
-    {.emit: """
-      var isNode = typeof module !== 'undefined' && module.exports
-
-      if (isNode) {
```

Comment:
Instead of detecting `nodejs` at runtime, now detect it at compile time.

Reply:
great; this matters because we shouldn't waste cycles at runtime for something that measures time